lib/browser/webapis-media-query.ts

```diff
@@ -23,16 +22,14 @@ import {NestedEventListenerOrEventListenerObject, patchEventTargetMethods} from
             handler: args[0],
             target: self || _global,
             name: 'mediaQuery',
-            invokeAddFunc: function(
-                addFnSymbol: any, delegate: Task|NestedEventListenerOrEventListenerObject) {
+            invokeAddFunc: function(addFnSymbol: any, delegate) {
```

Comment:
The reason is I don't want to import the type from until.ts which will drag all contents from util.ts when rollup build.